python/ray/nightly-wheels.yaml

```diff
@@ -0,0 +1,14 @@
+linux:
```

Comment:
Why not just bake this into the script? It seems a bit heavyweight to put it in its own file.

Reply:
I didn't think it was appropriate to place these hyperlinks/configuration in our scripts.py entrypoint.

IMO this is like a requirements.txt file, and I don't see a downside to doing it like this. It is also accessible should we want to validate/test/access this later (though I agree this isn't important). 

If you feel strongly, I can change it...